### PR TITLE
Change "Altom" in name for C# package with "Alttester"

### LINKED ISSUE #984: Change "Altom" in name for C# package with "Alttester"
## Description
 
e.g.
`using Altom.AltDriver.Commands` should become `using AltTester.AltDriver.Commands`

### PR DESCRIPTION
**Pull request by SandyEma9**
Closes #984

### CODE REVIEW

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15 @iulianatodoran @ReichEdvin

**Comment by iulianatodoran** [comment]
- Observed that not only the names for C# were changed, but also for Python and Java in tests. Was this also the subject of this ticket?
image

- Here you need to change the name also:
image
image

- Here I'm not sure if it needs to change also?
image

File: Docs/source/pages/upgrade-guides/v-1-8-0.rst [hunk 1]
```diff
@@ -72,7 +72,7 @@ Replace:
     <PropertyGroup>
         <PackageId>AltUnityDriver</PackageId>
         <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
-        <RootNamespace>Altom.AltUnityDriver</RootNamespace>
```
Comment (iulianatodoran) [inline Docs/source/pages/upgrade-guides/v-1-8-0.rst:75]: this was correct, it's in the `Replace` section

File: Docs/source/pages/upgrade-guides/v-1-8-0.rst [hunk 2]
```diff
@@ -72,7 +72,7 @@ Replace:
...
+        <RootNamespace>AltTester.AltUnityDriver</RootNamespace>
```
Comment (iulianatodoran) [inline Docs/source/pages/upgrade-guides/v-1-8-0.rst:75]: see my previous comment

Comment (iulianatodoran) [reply Docs/source/pages/upgrade-guides/v-1-8-0.rst:75]: the change needs to be made in the `With` section, not the `Replace` one, as I highlighted with a Red pen on my first comment thread with screenshots

**Review by iulianatodoran — APPROVED** [review]
LGTM

**Pull request merged**